source/sass/2017/markdown/code.scss

```diff
@@ -1,3 +1,12 @@
+.MarkdownBody {
+  & h2, & h3, & p, & ol, & ul {
+    code {
```

Comment:
I think the style here will be overridden by the style in line 48?
![image](https://user-images.githubusercontent.com/22890026/78607892-c64f8280-7814-11ea-8ca6-d2a9805abbce.png)